threshold.js

```diff
@@ -304,6 +317,20 @@ const experiment = (blockCount) => {
       opacity: 1.0,
       depth: -7.0,
     });
+    stepCount = new visual.TextStim({
```

Comment:
Structure-wise I don't suggest breaking code of flankers and target apart and they are one unit.